metric/doc.go

```diff
@@ -53,21 +53,54 @@ categories to use.
 
 Outside of these two broad categories, instruments are described by the
 function they are designed to serve. All Counters ([Int64Counter],
-[Float64Counter], [Int64ObservableCounter], [Float64ObservableCounter]) are
+[Float64Counter], [Int64ObservableCounter], and [Float64ObservableCounter]) are
 designed to measure values that never decrease in value, but instead only
 incrementally increase in value. UpDownCounters ([Int64UpDownCounter],
-[Float64UpDownCounter], [Int64ObservableUpDownCounter],
+[Float64UpDownCounter], [Int64ObservableUpDownCounter], and
 [Float64ObservableUpDownCounter]) on the other hand, are designed to measure
-values that can increase and decrease. When more information
-needs to be conveyed about all the synchronous measurements made during a
-collection cycle, a Histogram ([Int64Histogram], [Float64Histogram]) should be
-used. Finally, when just the most recent measurement needs to be conveyed about an
-asynchronous measurement, a Gauge ([Int64ObservableGauge],
+values that can increase and decrease. When more information needs to be
+conveyed about all the synchronous measurements made during a collection cycle,
+a Histogram ([Int64Histogram] and [Float64Histogram]) should be used. Finally,
+when just the most recent measurement needs to be conveyed about an
+asynchronous measurement, a Gauge ([Int64ObservableGauge] and
 [Float64ObservableGauge]) should be used.
 
 See the [OpenTelemetry documentation] for more information about instruments
 and their intended use.
 
+# Measurements
+
+Measurements are made by recording values and information about the values with
+an instrument. How these measurements are recorded depends on the instrument.
+
+Measurements for synchronous instruments ([Int64Counter], [Int64UpDownCounter],
+[Int64Histogram], [Float64Counter], [Float64UpDownCounter], and
+[Float64Histogram]) are recorded using the instrument methods directly. All
+counter instruments have an `Add` method that is used to measure an increment
```

Comment:
Here and on the next line, we surround method names with backticks. Per golang/go#41958, it sounds like the use here isn't a supported style hint, though that's not using balanced backticks; it's using a backtick or grave accent followed by an apostrophe.

Reply:
Ah, good catch. Updated.